mappings/net/minecraft/entity/ai/brain/task/ForgetCompletedPointOfInterestTask.mapping

```diff
@@ -4,3 +4,6 @@ CLASS net/minecraft/unmapped/C_ydgrguwg net/minecraft/entity/ai/brain/task/Forge
 		ARG 0 world
 		ARG 1 pos
 		ARG 2 entity
+	METHOD m_zavkqnbm tun (Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_vbbyoqyw;)Lnet/minecraft/unmapped/C_mdnathub;
```

Comment:
```suggestion
	METHOD m_zavkqnbm run (Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_vbbyoqyw;)Lnet/minecraft/unmapped/C_mdnathub;
```
I spent longer than I'd like to admit trying to figure out if this was some AI term I didn't know about before seeing `run` in the next class

Reply:
ugh they look so similar in the font, fixing this.